src/components/PopularRecipes/PopularRecipes.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from "react";
+import { getPopularRecipes } from "../../services/recipes";
+import styles from "./PopularRecipes.module.css";
+import RecipeCardContainer from "../RecipeCardContainer/RecipeCardContainer";
+import { Typography } from "../Typography/Typography";
+
+const PopularRecipes = () => {
+  const [recipes, setRecipes] = useState([]);
+
+  useEffect(() => {
+    const fetchPopular = async () => {
+      try {
+        const res = await getPopularRecipes();
+        const popularData = res.data.data.popularRecipes || [];
+
+        const adapted = popularData.map((item) => ({
+          ...item,
+          preview: item.thumb,
+          user: {
+            id: item.owner?.id,
+            name: item.owner?.name,
+            avatar: item.owner?.avatarURL?.startsWith("http")
+              ? item.owner.avatarURL
```

Comment:
You don't need this. Avatar and image components already do it. Just pass the data as it is to the PopularRecipes component